src/app/search-bar/search-bar.component.ts

```diff
@@ -67,8 +76,15 @@ export class SearchBarComponent implements OnInit, AfterViewInit {
     let instantsearch = JSON.parse(localStorage.getItem('instantsearch'));
 
     if (instantsearch && instantsearch.value) {
-      this.store.dispatch(new queryactions.QueryServerAction({'query': event, start: 0, rows: this.searchdata.rows}));
-      this.displayStatus = 'showbox';
+      if (this.searchdata.fq !== '') {
+        this.store.dispatch(new queryactions.QueryServerAction({'query': event, start: 0, rows: this.searchdata.rows, fq: this.searchdata.fq}));
+      } else {
+        this.store.dispatch(new queryactions.QueryServerAction({
+          'query': event,
```

Comment:
we need it to check if the query has the fq parameter, the presence of that parameter, will mean images need to be displayed, and the url is changed accordingly. Start and rows are present no matter what, fq is optional, needs to be checked first